cmake/Findnl-3.cmake

```diff
@@ -0,0 +1,23 @@
+find_package(PkgConfig REQUIRED)
+
+find_library(NL3_LIBRARY "nl-3")
+if(NOT NL3_LIBRARY)
+	message(FATAL_ERROR "libnl-3 not found!")
+endif()
+
+pkg_check_modules(LIBNL3 libnl-3.0 REQUIRED)
+
+add_library(nl-3 SHARED IMPORTED)
+
+if (TARGET_PLATFORM STREQUAL "linux")
```

Comment:
Please repeat the big comment why this should only be done on Linux. BTW, I think this is wrong, the -isystem should only be applied to elements of the [INTERFACE_SYSTEM_INCLUDE_DIRECTORIES](https://cmake.org/cmake/help/v3.0/prop_tgt/INTERFACE_SYSTEM_INCLUDE_DIRECTORIES.html) property. But that's a separate concern.

Reply:
added the comment

Reply:
will delete this file as it is not used